### PR TITLE
Feature/3318 drawer sub header api

### PR DESCRIPTION
**Pull request by ektasawant**
Fixes 3318 .

#### Changes proposed in this Pull Request:
- api docs for drawer sub header
- 

#### To Test:
- http://localhost:3000/components/drawer/drawer-sub-header/api-docs

### CODE REVIEW

**Review by huayunh — APPROVED** [review]

**Comment by huayunh** [comment]
@bkarambe , can you do something to make this gray area pure white? Looks like it was not addressed in #471 

image

**Review by jeffvg — APPROVED** [review]
looks good, agree with @huayunh about footer

File: docs/src/components/navigation/componentPreviewTabs.tsx [hunk 1]
```diff
@@ -73,7 +73,7 @@ const tabStyles = {
 const tabPanelContentStyles = {
     display: 'flex',
     flexDirection: 'column',
-    height: '100%',
+    height: (theme: Theme) => `calc(100vh - ${theme.spacing(8)})`,
```
Comment (huayunh) [inline docs/src/components/navigation/componentPreviewTabs.tsx:76]: Height shall stay 100%. 

The thing to change should be minHeight, or else you get strange things like this:
 
image

Comment (ektasawant) [reply docs/src/components/navigation/componentPreviewTabs.tsx]: > Height shall stay 100%.
> 
> The thing to change should be minHeight, or else you get strange things like this:
> 
> image

Thanks for this suggestion :). Done the changes also updated other readme with material UI URL.

**Pull request merged**